src/WCSimPMTObject.cc

```diff
@@ -1634,3 +1634,332 @@ G4float BoxandLine12inchHQE::GetDarkRateConversionFactor(){
+      0.998065, 0.998104, 0.998128, 0.998153, 0.998179,
+      0.998205, 0.998223, 0.998254, 0.998293, 0.998319,
+      0.998346, 0.998374, 0.998397, 0.998414, 0.998432,
+      0.998456, 0.998482, 0.998511, 0.998532, 0.998553,
+      0.998571, 0.998594, 0.998614, 0.998638, 0.998669,
+      0.998693, 0.998715, 0.998743, 0.998762, 0.998793,
+      0.998812, 0.998834, 0.998857, 0.998872, 0.998888,
+      0.998904, 0.998926, 0.998946, 0.998963, 0.998983,
+      // 7
+      0.999007, 0.999027, 0.999044, 0.999064, 0.999079,
+      0.999096, 0.999120, 0.999133, 0.999152, 0.999160,
+      0.999174, 0.999188, 0.999206, 0.999221, 0.999234,
+      0.999248, 0.999263, 0.999276, 0.999286, 0.999300,
+      0.999313, 0.999321, 0.999331, 0.999347, 0.999356,
+      0.999369, 0.999381, 0.999394, 0.999402, 0.999415,
+      0.999427, 0.999433, 0.999446, 0.999458, 0.999472,
+      0.999484, 0.999499, 0.999513, 0.999522, 0.999532,
+      0.999540, 0.999550, 0.999559, 0.999567, 0.999574,
+      0.999588, 0.999599, 0.999613, 0.999618, 0.999627,
+      // 8
+      0.999635, 0.999639, 0.999652, 0.999662, 0.999667,
+      0.999671, 0.999678, 0.999682, 0.999688, 0.999693,
+      0.999698, 0.999701, 0.999706, 0.999711, 0.999718,
+      0.999722, 0.999727, 0.999732, 0.999737, 0.999740,
+      0.999746, 0.999750, 0.999754, 0.999763, 0.999766,
+      0.999769, 0.999774, 0.999780, 0.999784, 0.999788,
+      0.999796, 0.999803, 0.999807, 0.999809, 0.999815,
+      0.999820, 0.999827, 0.999830, 0.999833, 0.999833,
+      0.999836, 0.999839, 0.999842, 0.999845, 0.999850,
+      0.999853, 0.999857, 0.999860, 0.999865, 0.999870,
+      // 9
+      0.999873, 0.999877, 0.999880, 0.999882, 0.999883,
+      0.999886, 0.999888, 0.999889, 0.999895, 0.999896,
+      0.999897, 0.999901, 0.999902, 0.999905, 0.999907,
+      0.999907, 0.999909, 0.999911, 0.999911, 0.999912,
+      0.999913, 0.999914, 0.999917, 0.999919, 0.999921,
+      0.999923, 0.999927, 0.999929, 0.999931, 0.999933,
+      0.999936, 0.999942, 0.999942, 0.999944, 0.999947,
+      0.999947, 0.999948, 0.999949, 0.999952, 0.999955,
+      0.999957, 0.999957, 0.999961, 0.999962, 0.999963,
+      0.999963, 0.999963, 0.999964, 0.999965, 0.999965,
+      // 10
+      0.999965, 0.999965, 0.999966, 0.999968, 0.999969,
+      0.999971, 0.999972, 0.999972, 0.999973, 0.999975,
+      0.999975, 0.999975, 0.999975, 0.999975, 0.999975,
+      0.999975, 0.999979, 0.999979, 0.999980, 0.999982,
+      0.999983, 0.999985, 0.999986, 0.999987, 0.999987,
+      0.999988, 0.999989, 0.999989, 0.999989, 0.999989,
+      0.999990, 0.999990, 0.999992, 0.999993, 0.999994,
+      0.999994, 0.999994, 0.999994, 0.999994, 0.999995,
+      0.999995, 0.999995, 0.999996, 0.999996, 0.999996,
+      0.999996, 0.999998, 0.999999, 1.000000, 1.000000,
+      // Dummy element for noticing if the loop reached the end of the array
+      0.0
+  };
+  return qpe0;
+}
+
+//Currenly the PMT QE info is the same as 20 inch.
```

Comment:
Ok, I see now that this QE isn't the same as the 20" PMT. So, you should remove this comment.